conversions/battery.js

```diff
@@ -0,0 +1,83 @@
+const Concentrate2 = require("concentrate2");
+const _ = require('lodash')
+
+module.exports = (app, plugin) => {
+
+  const batteryKeys = [
+    'voltage',
+    'current',
+    'temperature',
+    'capacity.stateOfCharge',
+    'capacity.timeRemaining',
+    'capacity.stateOfHealth',
+    'ripple'
+  ]
+  
+  return {
+    title: 'Battery (127506 & 127508)',
+    optionKey: 'BATTERY',
+    context: 'vessels.self',
+    properties: {
+      batteries: {
+        title: 'Battery Mapping',
+        type: 'array',
+        items: {
+          type: 'object',
+          properties: {
+            signalkId: {
+              title: 'Signal K battery id',
+              type: 'string'
+            },
+            instanceId: {
+              title: 'NMEA2000 Battery Instance Id',
+              type: 'number'
+            }
+          }
+        }
+      }
+    },
+
+    conversions: (options) => {
+      return options.BATTERY.batteries.map(battery => {
+        return {
+          keys: batteryKeys.map(key => `electrical.batteries.${battery.signalkId}.${key}`),
+          timeouts: batteryKeys.map(key => 60000),
```

Comment:
If all the timeouts are the same maybe a single `number` would do here and the calling code could handle arrays and single numbers. Pretty minor...